### DISCUSSION TITLE
Participating in Google Season of Docs

### DISCUSSION THREAD

**Discussion by CBID2** [open-sauced/feedback#28] [General feedback or Bugs]
Hi everyone. Google Season of Docs recently announced that they are accepting [applications from organizations](https://opensource.googleblog.com/2024/02/announcing-google-season-of-docs-2024.html). I think OpenSauced would benefit greatly from this program as it would help bring more contributors to the project. What do you think?

**Comment by bdougie** [comment]
@Bekah did we apply to this?

**Reply by BekahHW** [reply]
https://github.com/open-sauced/docs/wiki/Season-of-Docs-2024-Organization-Application:-OpenSauced
